src/tvheadend/entity/Tag.cpp

```diff
@@ -90,7 +90,7 @@ bool Tag::ContainsChannelType(bool bRadio) const
   {
     if ((cit = channels.find(*it)) != channels.end())
     {
-      if (bRadio == cit->second.IsRadio())
+      if (cit->second.GetType() == (bRadio ? CHANNEL_TYPE_RADIO : CHANNEL_TYPE_TV))
```

Comment:
Shouldn't <code>Tag::ContainsChannelType(bool bRadio)</code> consequently be changed to <code>Tag::ContainsChannelType(channel_type_t eType)? Currently implmentation returns true for "other" channels if bRadio is false which is not correct.